best-practices.md

```diff
@@ -147,19 +148,22 @@ For data providers using STAC with requester pays buckets, there are two main re
 
 ## Item Practices
 
-### Field and ID formatting
+### Item IDs
 
 When defining one's STAC properties and fields there are many choices to make on how to name various aspects of one's
 data. One of the key properties is the ID. The specification is quite flexible on ID's, primarily so that existing
 providers can easily use their same ID when they translate their data into STAC - they just need to be sure it is globally
-unique, so may need a prefix. But the use of URI reserved characters such as `:` or `/` is discouraged since this will 
+unique, so may need a prefix. But the use of URI or file path reserved characters such as `:` or `/` is discouraged since this will 
 result in [percented encoded](https://tools.ietf.org/html/rfc3986#section-2) [STAC API](https://github.com/radiantearth/stac-api-spec) 
-endpoints. This isn't a blocker, it just makes the ID's served through API's a bit less parsable. 
+endpoints and also the [IDs can't be used as file names](#catalog-layout).
```

Comment:
Not sure to understand. Does this mean it is not recommended to have an item with id `s1a_grd_20200912233421` in a file named `s1a_grd_20200912233421.json`?

Reply:
can we reword 
" and it prevents the [IDs to be used as file names](#catalog-layout)  "

Reply:
I think it's more saying that if you don't follow the recommendation to not use ':' or '/' then you can't follow the recommendation to use the item id.

Reply:
Yes, sure. What would you write instead?

> Does this mean it is not recommended to have an item with id s1a_grd_20200912233421 in a file named s1a_grd_20200912233421.json?

It is recommended to do exactly that.

Reply:
@cholmes Yes, I'd be happy if someone can re-phrase that to be easier to understand.

Reply:
```suggestion
endpoints and it prevents the use of IDs as file names as recommended in the [catalog layout](#catalog-layout) best practices.
```

Reply:
Sounds good!